src/heretic/mn_menu.c

```diff
@@ -1001,10 +1001,11 @@ static boolean SCMouseSensi(int option)
 {
     if (option == RIGHT_DIR)
     {
-        if (mouseSensitivity < 9)
```

Comment:
Please keep a reasonable limit. 255 ought to be enough for everyone. :wink:

Reply:
As in Doom:
https://github.com/fabiangreffrath/crispy-doom/blob/153d796a691cf2073e85ce997b0f7623a120492f/src/doom/m_menu.c#L1827